notebooks/117-model-server/117-model-server.ipynb

```diff
@@ -0,0 +1,549 @@
+    "│   ├── 1\n",
+    "│   │   ├── ir_model.bin\n",
+    "│   │   └── ir_model.xml\n",
+    "│   └── 2\n",
+    "│       ├── ir_model.bin\n",
+    "│       └── ir_model.xml\n",
+    "├── model2\n",
+    "│   └── 1\n",
+    "│       ├── ir_model.bin\n",
+    "│       ├── ir_model.xml\n",
+    "│       └── mapping_config.json\n",
+    "├── model3\n",
+    "│    └── 1\n",
+    "│        └── model.onnx\n",
+    "├── model4\n",
+    "│      └── 1\n",
+    "│        ├── model.pdiparams\n",
+    "│        └── model.pdmodel\n",
+    "└── model5\n",
+    "       └── 1\n",
+    "         └── TF_fronzen_model.pb\n",
+    "```\n",
+    "\n",
+    "\n",
+    "* Each model should be stored in a dedicated directory, e.g. model1 and model2.\n",
+    "\n",
+    "* Each model directory should include a sub-folder for each of its versions (1,2, etc). The versions and their folder names should be positive integer values.\n",
+    "\n",
+    "* Note: In execution, the versions are enabled according to a pre-defined version policy. If the client does not specify the version number in parameters, by default, the latest version is served.\n",
+    "\n",
+    "* Every version folder must include model files, that is, .bin and .xml for IR, .onnx for ONNX, .pdiparams and .pdmodel for Paddlepaddle. The file name can be arbitrary.\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d2a7a9a3",
+   "metadata": {},
+   "source": [
+    "### For Linux and MacOS user"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9230a63a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "path = Path(\"models/detection/1\")\n",
+    "if not path.exists():\n",
+    "    !mkdir -p models/detection/1\n",
+    "!cp ../004-hello-detection/model/* models/detection/1"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "382f1855",
+   "metadata": {},
+   "source": [
+    "### For Windows user"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "32681eb6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "path = Path(\"models/detection/1\")\n",
+    "if not path.exists():\n",
+    "    !mkdir -p models/detection/1\n",
+    "!mkdir models\\detection\\1\n",
+    "!copy ..\\004-hello-detection\\model\\* models\\detection\\1"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b66674df",
+   "metadata": {},
+   "source": [
+    "## Step 4: Start the Model Server Container\n",
+    "Start the container:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4fc2e171",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!docker run -d --rm  --name=\"ovms\" -v $(pwd)/models:/models -p 9000:9000 openvino/model_server:2022.3 \\\n",
+    "--model_path /models/detection/ --model_name detection --port 9000 --shape auto"
```

Comment:
Does the demo use reshaping feature? Is the input data the same as the model input? If yes, then `--shape auto` could be removed for simplicity.

Reply:
done